contracts/utils/Strings.sol

```diff
@@ -115,4 +120,85 @@ library Strings {
     function equal(string memory a, string memory b) internal pure returns (bool) {
         return bytes(a).length == bytes(b).length && keccak256(bytes(a)) == keccak256(bytes(b));
     }
+
+    /**
+     * @dev Parse an decimal string and returns the value as a `uint256`.
+     *
+     * This function will revert if:
+     * - the string contains any character that is not in [0-9].
+     * - the result does not fit in a uint256.
```

Comment:
```suggestion
     * @dev Parse a string in decimal (i.e. base 10) and returns the value as a `uint256`.
     *
     * This function will revert if:
     * - the string contains any character that is not in [0-9].
     * - the result does not fit in a `uint256`.
```

Reply:
When looking online, It looks like "decimal string" is the correct way. Maybe "string in decimal format" would be ok, but "string in decimal" doesn't look commonly used.

Reply:
https://books.google.com/ngrams/graph?content=decimal+string%2Cstring+in+decimal&year_start=1800&year_end=2019&corpus=en-2019&smoothing=0